### DISCUSSION TITLE
Generate Table Partitions

### DISCUSSION THREAD

**Discussion by TienNguyenVanDev** [GoogleCloudPlatform/professional-services-data-validator#1336] [Q&A] [answered]
Currently, I'm validating data between Oracle and PostgresSql, I have a table of big data of about 500M. 

How to generate a file yaml to validate 10k newest data?  I checked the document but I didn't find the solution to handle my case.

Please support me.
Thank you so much.

**Comment by helensilva14** [comment, marked as answer]
Hi @TienNguyenVanDev! We actually don't use this Community Discussions feature, we track questions as Github Issues. I created #1341 for us to discuss your scenario there and I'm closing this thread here. Thank you very much for contacting us!
